consensus/polybft/contracts_initializer.go

```diff
@@ -40,11 +40,11 @@ var (
 )
 
 func getInitChildValidatorSetInput(polyBFTConfig PolyBFTConfig) ([]byte, error) {
-	validatorAddresses := make([]types.Address, len(polyBFTConfig.InitialValidatorSet))
-	validatorPubkeys := make([][4]*big.Int, len(polyBFTConfig.InitialValidatorSet))
-	validatorStakes := make([]*big.Int, len(polyBFTConfig.InitialValidatorSet))
+	validatorAddresses := make([]types.Address, len(polyBFTConfig.Validators))
```

Comment:
No it wasn't, but I felt it is a good opportunity since I was changing genesis configuration layout anyways (adding rootchain contracts addresses to it, rootchain account address etc.). Do you agree?

Reply:
I agree with the change but not as part of this PR.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f35ba13ffe9fa37fd11178fe49e55f2f7c6f73d4